pkg/proc/eval.go

```diff
@@ -489,14 +490,19 @@ func (scope *EvalScope) simpleLocals(flags localsFlags, wantedName string) ([]*V
 				val.Flags |= VariableArgument
 			}
 		}
-		depths = append(depths, depth)
```

Comment:
I don't think we should keep around the depth value just so we can rewrite this particular sort.

Reply:
So, should I revert the changes to this file and leave `sort.Stable` as it is?

Reply:
Yes, I think so.

Reply:
PS. I benchmarked them back in the day, the speedup of slices.SortFunc over sort.Slice is basically nothing. If anything this should be refactored to sort.SliceStable from sort.Stable.

Reply:
Refactored to `sort.SliceStable`.